client/my-sites/add-ons/index.ts

```diff
@@ -0,0 +1,18 @@
+import page from 'page';
+import { makeLayout, render as clientRender } from 'calypso/controller';
+import { siteSelection, sites, navigation } from 'calypso/my-sites/controller';
+import { addOnsSiteSelectionHeader, addOnsManagement } from './controller';
+
+export default function () {
+	page( '/add-ons', siteSelection, addOnsSiteSelectionHeader, sites, makeLayout, clientRender );
+	page(
+		'/add-ons/:site',
+		siteSelection,
+		addOnsSiteSelectionHeader,
+		sites,
```

Comment:
To fix the section CSS classes, you need to remove this `sites` handler from the route definition.

It's rendering a site selection UI, which resets all the section-specific styling. And you never _actually_ render the site selection UI because the `addOnsManagement` will override the content with the add-ons UI.

What you want is to define a route without a `:site` parameter where the site selection UI would be:
```js
page( '/add-ons',
  siteSelection, // resets selected site ID to null
  sites, // requests render of the site selection UI
  makeLayout, // actually renders the UI with React
  clientRender
);